accepted/0000-dashboard.md

```diff
@@ -0,0 +1,132 @@
+# Radiate Buildpack and Community Metadata via a Dashboard
+
+## Summary
+<!-- A concise, one-paragraph description of the change. -->
+
+The [Paketo website](https://paketo.io) should include a new subdomain
+(`https://dashboard.paketo.io`) that displays details about the current status
+of buildpacks and other community information. This type of metadata helps the
+core development team keep on top of open issues and PRs while also making that
+information transparent for external community members.
+
+## Motivation
+<!--
+Why are we doing this? What pain points does this resolve? What use cases does
+it support? What is the expected outcome? Use real, concrete examples to make
+your case!
+-->
+
+While GitHub notifications and search can provide some amount of insight into
+the overall health and state of the Paketo buildpack orgs, finding answers to
+many questions is not a simple task.  I personally have found myself with
+multiple tabs pinned in my browser showing a number of different details about
+the state of buildpacks across the orgs. I'd like to know:
```

Comment:
There was some discussion in the past of having some sort of tool to track the time it takes from when a high/medium/low CVE is resolved upstream by Canonical to releasing a patched buildpack/stack. Seems like this dashboard would be a good place to surface this information (once it's built). cc @paketo-buildpacks/core-dependencies